ci-operator/README.md

```diff
@@ -294,7 +294,7 @@ With the template file ready, the steps required to add it to the repository
 and make it available for CI jobs are:
 
 1. Create the yaml file in the `templates/` directory.
-2. Add the files to the `config-updater` section of Prow's configuration file
+2. Add the files to the [`config-updater` section of Prow's configuration file](https://github.com/openshift/release/blob/master/cluster/ci/config/prow/plugins.yaml)
```

Comment:
Can this be a relative link?

    ... configuration file](../cluster/ci/config/prow/plugins.yaml)

Reply:
I'm going to leave it as/is, bc it goes along with the other links in the doc.